src/libraries/Microsoft.Extensions.Logging.Console/src/AnsiLogConsole.cs

```diff
@@ -21,106 +21,15 @@ public AnsiLogConsole(IAnsiSystemConsole systemConsole)
             _systemConsole = systemConsole;
         }
 
-        public void Write(string message, ConsoleColor? background, ConsoleColor? foreground)
+        public void Write(string message)
         {
-            // Order: backgroundcolor, foregroundcolor, Message, reset foregroundcolor, reset backgroundcolor
-            if (background.HasValue)
-            {
-                _outputBuilder.Append(GetBackgroundColorEscapeCode(background.Value));
-            }
-
-            if (foreground.HasValue)
-            {
-                _outputBuilder.Append(GetForegroundColorEscapeCode(foreground.Value));
-            }
-
             _outputBuilder.Append(message);
```

Comment:
Since we are no longer doing any processing, we can just write directly to the Console here. And that means we can remove the "Flush()" method all together, since it won't do anything for either implementation.

Reply:
Are you recommending Flush be dropped from IConsole?
Because both AnsiLogConsole and WindowsLogConsole are implementing Flush as part of IConsole interface. AnsiLogConsole has implementation for Flush.